codebuild/bin/install_awslc_fips.sh

```diff
@@ -50,12 +51,17 @@ INSTALL_DIR=$2
 VERSION=$3
 
 # Map version to a specific feature branch/tag.
+# Note: since the next FIPS validation will be split off from main
+# building main with FIPS enabled is essentially the next FIPS release.
 case $VERSION in
   "2022")
-    AWSLC_BRANCH=AWS-LC-FIPS-2.0.17
+    AWSLC_BRANCH=fips-2022-11-02
     ;;
   "2024")
```

Comment:
Do we plan to change these names? Like have a fixed "previous" and "current"? I recognize that'll require updating a bunch of buildspecs :)

Reply:
I was hoping to do that in a followup PR...